wiki/osu!wiki_contribution_guide/en.md

```diff
@@ -1,178 +1,817 @@
+- Basic knowledge in VCS (Version control system), particularly in using [Git](https://git-scm.com/ "Git").
+
+## Forking osu-wiki repository
 
-## Forking
- 
- To be able to make changes to the repository, you must first create your own copy of the repository. This is known as "forking".
- 
- Go to the [osu!wiki repository](https://github.com/ppy/osu-wiki).
- 
- You should see something like this.
+To make changes to the osu-wiki repository, you must first create your own copy of the repository.
+This is known as "forking".
 
-![osu-wiki Repository Snapshot](3.jpg "Snapshot of the osu!Wiki Repository")
+Go to the [osu!wiki repository](https://github.com/ppy/osu-wiki "osu-wiki in GitHub").
 
-Click on the "Fork" button, located at the top-right of the page. This should redirect you to the location for the fork.
+You should see something like this.
 
-![Fork prompt dialogue](4.jpg "Fork prompt dialogue")
+![osu-wiki Repository Snapshot](Contribute_wikirepo.jpg "Snapshot of the osu-wiki repository")
 
+Click on the "Fork" button, located at the top-right of the page.
+This should redirect you to the location for the fork.
+
+![Fork prompt dialogue](Contribute_forkprompt.jpg "Fork prompt dialogue")
 
 Now click at your profile and GitHub would redirect you to where the fork is.
 
 Finally, you should see something like this.
 
-![Fork Snapshot](5.jpg "Fork Snapshot")
+![Fork Snapshot](Contribute_ownrepo.jpg "Fork Snapshot")
+
+Congratulations!
+You have made a copy of the osu-wiki repository!
 
-Congratulations! You have made a copy of the osu!wiki repository! You can now proceed to make changes.
+You can now proceed to make changes.
 
-## Making Changes 
+## Making Changes
 
 ### Through GitHub's Web Editor
 
-If you don't want the hassle of cloning your fork into your PC, GitHub offers editing through your browser.
+#### Accessing the correct repository and branch.
 
-To edit a page, go to the folder of that page and select the language of that article. From here, you should see something like this.
+If you don't want the hassle of cloning your fork into your local machine, GitHub offers a web editor for editing through your browser.
 
-_Note: Each article is grouped into folders where they contain markdown files arranged by language and their media. All article pages inside these folders must use the two letter language codes. To know which language code to use, see the [List of ISO 639-1 codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) following the `639-1` column. Please follow this format when creating a new article._                  
+First, visit your own forked repo (you can rename the ``ppy`` part in ``https://github.com/ppy/osu-wiki`` with your own GitHub's name for quicker access or click on the "Fork" button in ``ppy/osu-wiki``).
 
+You should be in the ``master`` branch by default.
 
-![GitHub Preview](7.jpg "Github Preview")
+#### Branching
```

Comment:
the name of this section sounds confusing, I mean, you should also branch when making local changes.